client/src/test/java/com/sinch/sdk/SinchClientTest.java

```diff
@@ -57,4 +59,48 @@ void defaultSmsUrlAvailable() {
     SinchClient client = new SinchClient(configuration);
     assertNotNull(client.getConfiguration().getSmsUrl());
   }
+
+  @Test
+  void defaultVerificationUrlAvailable() {
+    Configuration configuration =
+        Configuration.builder().setKeyId("foo").setKeySecret("foo").setProjectId("foo").build();
+    SinchClient client = new SinchClient(configuration);
+    assertNotNull(client.getConfiguration().getVerificationUrl());
+  }
+
+  @Test
+  void defaultVoiceUrl() {
+    Configuration configuration =
+        Configuration.builder().setKeyId("foo").setKeySecret("foo").setProjectId("foo").build();
```

Comment:
About the credentials management, which direction did you choose finally? Because here I see that the project Id + access key and secret are used to initialize the configuration object although these credentials are not required for using the voice API alone

Reply:
Yes.
Currently waiting for ADR validation for credentials
We do not have choice to going to be less restrictive (credentials by domains) just we have to be consistent across SDKs

Reply:
Ok, let's wait for the meeting on Jan 10th then. Wow that's next year!!